### PR TITLE
#11942: Restoring JAVA_OPTS

### PR DESCRIPTION
**Pull request by danlaguna**

### CODE REVIEW

**Review by jdotcms — APPROVED** [review]

**Review by dsilvam — APPROVED** [review]

File: bin/startup.sh [hunk 1]
```diff
@@ -57,7 +57,18 @@ DOTCMS_HOME=`cd "$PRGDIR/../$HOME_FOLDER" ; pwd`
 #JAVA_OPTS="-agentpath:/Applications/YourKit_Java_Profiler_9.0.5.app/bin/mac/libyjpagent.jnilib $JAVA_OPTS -Djava.awt.headless=true -Xverify:none -Dfile.encoding=UTF8 -server -Xms1024M -Xmx1024M -XX:PermSize=128m "
 
 #JAVA_OPTS="$JAVA_OPTS -Xdebug -Xrunjdwp:transport=dt_socket,address=8000,server=y,suspend=n"
-JAVA_OPTS="$JAVA_OPTS -Djava.awt.headless=true -Xverify:none -Dfile.encoding=UTF8 -Xmx1G -XX:+UseG1GC -javaagent:$DOTCMS_HOME/WEB-INF/lib/byte-buddy-agent-1.6.12.jar"
+
+#JAVA_OPTS="$JAVA_OPTS -Djava.awt.headless=true -Xverify:none -Dfile.encoding=UTF8 -Xmx1G -XX:+UseG1GC -javaagent:$DOTCMS_HOME/WEB-INF/lib/byte-buddy-agent-1.6.12.jar"
+JAVA_OPTS="$JAVA_OPTS -Djava.awt.headless=true -Xverify:none -Dfile.encoding=UTF8 -server -XX:+DisableExplicitGC -XX:+UseConcMarkSweepGC"
```
Comment (wezell) [inline bin/startup.sh]: I think we need to get rid of:

`-XX:+UseConcMarkSweepGC`

File: bin/startup.sh [hunk 2]
```diff
@@ -57,7 +57,18 @@ DOTCMS_HOME=`cd "$PRGDIR/../$HOME_FOLDER" ; pwd`
...
+
+# Set Memory sizing
+JAVA_OPTS="$JAVA_OPTS -XX:MaxMetaspaceSize=512m -Xmx1G"
+
+# Set GC opts
+JAVA_OPTS="$JAVA_OPTS -XX:+UseG1GC"
```
Comment (wezell) [inline bin/startup.sh]: we use G1gc instead

File: bin/startup.sh (hunk 1 above)
Comment (danlaguna) [inline bin/startup.sh]: I will remove this prop

File: bin/startup.sh (hunk 2 above)
Comment (danlaguna) [inline bin/startup.sh]: How should this entry look like? JAVA_OPTS="$JAVA_OPTS -XX:+**UseG1gc**"

**Pull request merged**